documentation/ztp-for-factories/ztp-for-factory-edgeclusters-yaml.adoc

```diff
@@ -11,9 +11,7 @@ xref:edgeclusters[edgeclusters]:
       xref:mastername[master0]:
         xref:ignore_ifaces[ignore_ifaces]: eno1 eno2
         xref:nic_ext_dhcp[nic_ext_dhcp]: eno4
-        xref:nic_int_static[nic_int_static]: eno5
```

Comment:
If this is a specific use case, I would *EITHER* duplicate the documentation or just add Optional, the standard use case so far has been to use double NICS, removing them from the docs might confuse users (this applies to all the docs and the example  `config.yaml`).

Reply:
TBH my idea is just to add the example with the 2 possibilities:
edgecluster0 -> with only 1 nic
edgecluster1 -> with 2 nics 
The same with the config.yml in the example folder...
Makes sense for you? WDYT?

Reply:
I would suggest to add a note before the config file if you don't want to put 'optional' on each one to show that there are different examples

Reply:
There is a note before as well as in the main section in the network requirements. (https://github.com/rh-ecosystem-edge/ztp-pipeline-relocatable/pull/348/files#diff-e34d565cdbd7cccb59c67202350f20cef1f25a830511b14d2dd5115bdc9573d3). I mean there're notes in all points involved in the networking but adding this (optional) to the diagram and to the example make it clear

Reply:
@iranzo Do you think is not clear in the PR??